runtime/westend/src/lib.rs

```diff
@@ -1835,3 +1859,42 @@ mod remote_tests {
 		ext.execute_with(|| Runtime::on_runtime_upgrade());
 	}
 }
+
+mod init_state_migration {
+	use super::Runtime;
+	use frame_support::traits::OnRuntimeUpgrade;
+	use pallet_state_trie_migration::{AutoLimits, MigrationLimits, MigrationProcess};
+	#[cfg(not(feature = "std"))]
+	use sp_std::prelude::*;
+
+	/// Initialize an automatic migration process.
+	pub struct InitMigrate;
+	impl OnRuntimeUpgrade for InitMigrate {
+		#[cfg(feature = "try-runtime")]
+		fn pre_upgrade() -> Result<Vec<u8>, &'static str> {
+			Ok(Default::default())
```

Comment:
You can abort here if the `AutoLimits` value already exists, so we wont forget to remove the migration.